sound/smooth_swing_v2.h

```diff
@@ -7,220 +7,254 @@
 //
 class SmoothSwingV2 : public SaberBasePassThrough {
 public:
-  SmoothSwingV2() : SaberBasePassThrough() {}
-
-  void Activate(SaberBase* base_font) {
-    STDOUT.println("Activating SmoothSwing V2");
-    SetDelegate(base_font);
-    if (swingl.files_found() != swingh.files_found()) {
-      STDOUT.println("Warning, swingl and swingh should have the same number of files.");
-    }
-    swings_ = std::min<size_t>(swingl.files_found(), swingh.files_found());
-  }
-
-  void Deactivate() {
-    SetDelegate(NULL);
-    A.Free();
-    B.Free();
-  }
-
-  void Swap() {
-    Data C = A;
-    A = B;
-    B = C;
-  }
-
-  // Should only be done when the volume is near zero.
-  void PickRandomSwing() {
-    if (!on_) return;
-    uint32_t m = millis();
-    // No point in picking a new random so soon after picking one.
-    if (A.player && m - last_random_ < 1000) return;
-    last_random_ = m;
-    int swing = random(swings_);
-    float start = m / 1000.0;
-    A.Stop();
-    B.Stop();
-    swingl.Select(swing);
-    swingh.Select(swing);
-    A.Play(&swingl, start);
-    B.Play(&swingh, start);
-    if (random(2)) Swap();
-    float t1_offset = random(1000) / 1000.0 * 50 + 10;
-    A.SetTransition(t1_offset, smooth_swing_config.Transition1Degrees);
-    B.SetTransition(t1_offset + 180.0,
-      smooth_swing_config.Transition2Degrees);
-  }
-
-  void SB_On() override {
-    on_ = true;
-    // Starts hum, etc.
-    delegate_->SB_On();
-    PickRandomSwing();
-    if (!A.player || !B.player) {
-      STDOUT.println("SmoothSwing V2 cannot allocate wav player.");
-    }
-  }
-  void SB_Off() override {
-    on_ = false;
-    A.Off();
-    B.Off();
-    delegate_->SB_Off();
-  }
-
-  enum class SwingState {
-    OFF, // waiting for swing to start
-    ON,  // swinging
-    OUT, // Waiting for sound to fade out
-  };
-
-  void SB_Motion(const Vec3& raw_gyro, bool clear) override {
-    if (clear) {
-      gyro_filter_.filter(raw_gyro);
-      gyro_filter_.filter(raw_gyro);
-    }
-    Vec3 gyro = gyro_filter_.filter(raw_gyro);
-    // degrees per second
-    // May not need to smooth gyro since volume is smoothed.
-    float speed = sqrtf(gyro.z * gyro.z + gyro.y * gyro.y);
-    uint32_t t = micros();
-    uint32_t delta = t - last_micros_;
-    if (delta > 1000000) delta = 1;
-    last_micros_ = t;
-    float hum_volume = 1.0;
+    SmoothSwingV2() : SaberBasePassThrough() {}
```

Comment:
It's impossible to see what is changed, because it shows every line in the file as changed for some reason.
Did you change the line endings or re-indent or something?

Reply:
Yeah, I must have.  Let me see if I can clean it up...